lib/rmt/checksum_verifier.rb

```diff
@@ -0,0 +1,32 @@
+class RMT::ChecksumVerifier
+
+  KNOWN_HASH_FUNCTIONS = %i[MD5 SHA1 SHA256 SHA384 SHA512].freeze
+
+  class Exception < RuntimeError
+  end
+
+  ##
+  # This method verifies that files are what they say that they are.
```

Comment:
I'm not sure how helpful that description is (and if it is needed at all). The method name is `verify_checksum`, it would have been weird if it did anything else :grimacing: